packages/marshal/src/marshal.js

```diff
@@ -1027,7 +1061,8 @@ export function makeMarshal(
         }
         return ibidTable.finish(result);
       }
-    };
+    }
+    return fullRevive;
```

Comment:
Is there a fall through case? Is this dead code?

Reply:
Not dead.  It's returning the function that is defined above.

Reply:
Doh!